.github/.ubiquibot-config.yml

```diff
@@ -0,0 +1,149 @@
+                      img: 5
+                      strong: 0
+                      blockquote: 0
+                      h1: 1
+                      h2: 1
+                      h3: 1
+                      h4: 1
+                      h5: 1
+                      h6: 1
+                      a: 5
+                      li: 1
+                      td: 1
+                      hr: 0
+                - role: [ ISSUE_AUTHOR ]
+                  multiplier: 1
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.2
+                - role: [ ISSUE_ASSIGNEE ]
+                  multiplier: 1
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.1
+                - role: [ ISSUE_COLLABORATOR ]
+                  multiplier: 1
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.1
+                - role: [ ISSUE_CONTRIBUTOR ]
+                  multiplier: 0.25
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.1
+                - role: [ PULL_SPECIFICATION ]
+                  multiplier: 0
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0
+                - role: [ PULL_AUTHOR ]
+                  multiplier: 0
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.2
+                - role: [ PULL_ASSIGNEE ]
+                  multiplier: 1
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.1
+                - role: [ PULL_COLLABORATOR ]
+                  multiplier: 1
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.1
+                - role: [ PULL_CONTRIBUTOR ]
+                  multiplier: 0.25
+                  rewards:
+                    regex:
+                      "\\b\\w+\\b": 0.1
+            permitGeneration:
+              enabled: true
+            githubComment:
+              post: true
+              debug: false
+  - uses:
+    - plugin: ubiquibot/user-activity-watcher@development
+      with:
+        watch:
+          optOut:
+            - ubiquibot
+            - launch-party
+            - staging
+            - production
+  - uses:
+    - plugin: ubiquibot/automated-merging@development
+      with:
+        approvalsRequired:
+          collaborator: 1
+        mergeTimeout:
+          collaborator: "3.5 days"
+        repos:
+          ignore:
+            - ubiquibot
+            - launch-party
+            - staging
+            - production
+  - uses:
+    - plugin: https://ubiquibot-command-start-stop-development.ubiquity.workers.dev
+      with:
+        reviewDelayTolerance: "3 Days"
+        taskStaleTimeoutDuration: "30 Days"
+        startRequiresWallet: true # default is true
+  - uses:
+    - plugin: ubiquibot/issue-comment-embeddings@development
```

Comment:
https://github.com/ubiquibot/ubiquibot-config/blob/6dc55a22c7cdafda06b75f87b6fd96c04ce5c40c/.github/.ubiquibot-config.yml#L3

Is this correct because the prod config is using my worker for comment embeddings

Reply:
`ubiquity` and `ubiquibot` organizations use different config setups for this plugin:
1. https://github.com/ubiquity/ubiquibot-config/blob/b77dd81838b133c7b4a4332c6c8e0db14d96a4ad/.github/.ubiquibot-config.yml#L152
2. https://github.com/ubiquibot/ubiquibot-config/blob/c602b2756bddc28f406b7adb26c42d4d5a211393/.github/.ubiquibot-config.yml#L3

What is the correct one?

Reply:
As far as I'm aware below is correct mate. I'll take the heat for not installing the worker into `Ubiquity` that was my fault it slipped my mind that we have two configs

```yml
plugins:
  - uses:
    - plugin: https://ubiquity-os-comment-vector-embeddings.keyrxng7749.workers.dev
```

Reply:
Although the repo is being targeted and the workflows are failing atm. Which I don't understand how because the repo name is different from what is in both configs

https://github.com/ubiquibot/generate-vector-embeddings/actions/workflows/compute.yml

Reply:
I didn't know that but yeah I agree.

I'm also updating my worker instance with the latest from that repo right now

Reply:
Looks like it never worked. Just scrolled to the beginning. 

https://github.com/ubiquibot/generate-vector-embeddings/actions/runs/10761301187

Reply:
@Keyrxng So we should use this config everywhere:
```
plugins:
  - uses:
    - plugin: https://ubiquity-os-comment-vector-embeddings.keyrxng7749.workers.dev
```

Correct?

Reply:
Well it's about to be refactored a little, but I guess if it's not supposed to be a worker it can target the official repo via workflows. I thought it was supposed to be a worker which is why I deployed mine.

Reply:
> Well it's about to be refactored a little, but I guess if it's not supposed to be a worker it can target the official repo via workflows. I thought it was supposed to be a worker which is why I deployed mine.

So in the end [this](https://github.com/ubiquity/ubiquibot-config/blob/b77dd81838b133c7b4a4332c6c8e0db14d96a4ad/.github/.ubiquibot-config.yml#L152) setting is correct, right?

Reply:
Yeah my bad the confusion was mine because I was running the only working instance as a worker so this is right sorry @rndquu